src/redux/users-reducer.js

```diff
@@ -24,7 +24,7 @@ const usersSlice = createSlice({
     reducers: {},
     extraReducers: {
         [updateUserThunk.fulfilled]: (state, action) => {
-            state.currentUser = action.payload;
+            // state.currentUser = action.payload;
```

Comment:
commented this out so that when i update the 2 user's following/followers list, the logged in user stays as currentUser. lmk if this affects anything! I can try swapping the order if this line is needed